src/tir/schedule/primitive/blockize_tensorize.cc

```diff
@@ -604,6 +604,15 @@ void Tensorize(ScheduleState self, const StmtSRef& sref, const TensorIntrin& int
     BlockNode* block = block_realize.CopyOnWrite()->block.CopyOnWrite();
     block->body = impl_block->body;
     block->match_buffers = std::move(match_buffer_regions);
+    for (auto [key, val] : impl_block->annotations) {
```

Comment:
nit: Can we use `auto&` instead of `auto`?

Reply:
updated